Cython/Compiler/FusedNode.py

```diff
@@ -293,8 +293,12 @@ def _fused_instance_checks(self, normal_types, pyx_code, env):
         """
         for specialized_type in normal_types:
             # all_numeric = all_numeric and specialized_type.is_numeric
+            py_type_name = specialized_type.py_type_name()
```

Comment:
This is my attempt to support int/long under Python 2. Does it look good enough?